i18n/locales/en.json

```diff
@@ -1204,21 +1204,21 @@
       "show": "Show advanced settings",
       "sync": "Sync editions to {0}"
     },
-    "landing": {
-      "collection": "Create A Collection For Your NFTs",
-      "heading": "What Do You Wish To Create?",
-      "massNft": "Multiple NFTs",
-      "massNftDesc": "Create and mint multiple different NFTs at once",
-      "nft": "Create awesome NFTs from your art and add them to your existing collection",
-      "singleNft": "Single NFT",
-      "singleNftDesc": "Create and mint a single original NFT (can have more copies)"
-    },
     "listForSale": "List NFT for sale",
     "listWithRoyalty": "Add royalty",
     "mass": "NFT mass minter",
     "mintCollectionSuccess": "Collection {name} Saved in block {block}",
     "mintNFTSuccess": "NFT {name} Saved in block {block}",
     "mintType": "Mint Type",
+    "modal": {
+      "availableOn": "Available on {0} and {1}",
+      "createCollection": "Create Collection",
+      "createCollectionDesc": "Organize your NFTs together",
+      "createMultiple": "Create Multiple NFTs",
+      "createMultipleDesc": "Batch creation in one transaction",
+      "createSingle": "Create Single NFT",
+      "createSingleDesc": "One-of-One, unique token"
+    },
     "nfsw": "Explicit content (NSFW)",
     "nfswMessage": "Set your collection as explicit and sensitive content.",
     "nft": {
```

Comment:
1. unrelated bug, label shows connect wallet but form is submitted

https://github.com/user-attachments/assets/05ab471c-cacf-4ab2-b4dc-aa67570b37e3

2. i think both create collection and create nft should have the same behaviour 

![CleanShot 2025-04-08 at 10 15 27@2x](https://github.com/user-attachments/assets/16e456f0-8fd7-4c6c-b919-0e7ce92913ac)

3. if i land on a create page without a connected wallet should the connect wallet modal open ?

Reply:
> 1 and 2 -> https://github.com/kodadot/nft-gallery/issues/11522
> if i land on a create page without a connected wallet should the connect wallet modal open ?

It's kind of disturb that automatically open a wallet modal when visit the create page. It's still related to the #11522
Ideally, showing the wallet modal when clicking the submit/connect button.

Reply:
hey there, something needs to be resolved from my side or?

Reply:
> hey there, something needs to be resolved from my side or?

Yes, please check #11522 and feel free to give us some input.

Reply:
> It's kind of disturb that automatically open a wallet modal when visit the create page. It's still related to the #11522

yes

> Ideally, showing the wallet modal when clicking the submit/connect button.

I agree